sqlx-core/src/sqlite/options/synchronous.rs

```diff
@@ -0,0 +1,49 @@
+use crate::error::Error;
+use std::str::FromStr;
+
+/// Refer to [SQLite documentation] for the meaning of various synchronous settings.
+///
+/// [SQLite documentation]: https://www.sqlite.org/pragma.html#pragma_synchronous
+#[derive(Debug, Clone)]
+pub enum SqliteSynchronousOption {
```

Comment:
Can we name this `SqliteSynchronous` instead of `SqliteSynchronousOption` ?

Reply:
I have been thinking about it. 
While `journal_mode=WAL` has no bad impact as default (other than running into `SQLITE_BUSY_SNAPSHOT` if used inappropriately), the `synchronous=NORMAL` setting might be unexpected.
People have mental model that when you commit it's down to the metal (on hard-drive) and if anything bad happen there is no data loss.
But it is also true that majority of application do not need this extra durability. A loss of couple of last writes is usually of no harm.

I have chosen an opt-in for performance instead of opt-in for durability.
What do you think about it ?

Note however that I mimicked `SqliteJournalMode`. Thus `Full` is also repeated explicitly in `SqliteConnectionOptions::new()`.
And I don't know if it's intended.

Reply:
done

Reply:
Sorry, I didn't push, now it's renamed for real.